docs/source/README.md

```diff
@@ -0,0 +1,2 @@
+# docs/source
+This directory will be used to contain the code of the docs of this SDK.
```

Comment:
We don't typically store any kind of docs in our repo like this.  Is this a Python thing? If you are wanting to do auto-generated docs like `javadoc`, then there is a different way to handle it.